kibana/fleet.go

```diff
@@ -628,6 +683,59 @@ func (client *Client) DeleteFleetPackage(ctx context.Context, packagePolicyID st
 	return r, err
 }
 
+//
+// Fleet Proxies
+//
+
+type ProxiesRequest struct {
```

Comment:
Should we add links to the docs like for the rest of the structs?

Reply:
Kinda, there isn't a standard in the file. So I created one, removing those redundant comments and linking to the API doc on the comment marking the methods about the API. Like [this](https://github.com/elastic/elastic-agent-libs/pull/246/files#diff-7e6d55d12ca5e0be223823cc8a7d5c460b540f9cafe565d1052b0a92e3fb39d8R654-R657).